test/unit_tests/user/UserRepositorySpec.coffee

```diff
@@ -254,20 +254,24 @@ describe 'z.user.UserRepository', ->
 
       it 'resolves with username when username is not taken', (done) ->
         usernames = ['john_doe']
-        server.respondWith 'GET', "#{test_factory.settings.connection.rest_url}/users?handles=#{usernames[0]}", [404, {}, '']
+        server.respondWith 'POST', "#{test_factory.settings.connection.rest_url}/users/handles", [
+          200,
+          {'Content-Type': 'application/json'},
```

Comment:
the brackets could be also go away.. but regardless of that the commas are not needed..